test/Frame_spec.js

```diff
@@ -30,6 +30,12 @@ describe("Frame test",function(){
     expect(frame.getDOMNode().style.border).toContain('0');
   });
 
+  it("should create an empty iFrame and apply given className", function () {
```

Comment:
Make make this more generic `should pass along all props to underlying iFrame` or something along those lines.

Then have a few expectations with different props.